guide/onboarding/manual-backup.md

```diff
@@ -11,9 +11,30 @@ main_classes: -no-top-padding
 
 # Manual backup
 
-### How it works
+## Backing up channel state
 
-It usually requires users to manually back up their 12 to 24 word recovery phrase by writing it down on a piece of paper and storing it in a safe (but memorable location). In the case that a user’s device breaks or is stolen, the user can recover their funds and wallet by correctly entering their recovery phrase. The [private key management]({{ '/guide/private-key-management/manual-backup/' | relative_url }}) section dives further into the technical details of this scheme if interested in learning more.
+A channel state backup is typically a file which includes information about the user's channel balances. This file can be downloaded onto the user's device and used to restore the Lightning channel state on another device. This could be a database of the most current channel balance information, or a static backup (as an absolute last resort).
+
+{% include tip/open.html color="red" icon="forbid" label="Manual channel state backups are for advanced users" %}
+
+Manual backups of Lightning channel state should only be performed with careful consideration. This is because the channel state backup may become outdated very quickly, and attempting to restore from an outdated backup could harm the user's funds. In the interest of adhering to the principles of [self-custody]({{ '/guide/designing-products/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/designing-products/principles/#transparency' | relative_url }}), you may choose to allow your user to perform a manual channel state backup. However, this is not recommended for the majority of users. If this option is available, keep it tucked away in a settings menu and prepend a warning that the option is only for advanced users.
+
+{% include tip/close.html %}
+
+## Backing up wallet recovery data
+
+Manual backup should be used primarily for wallet recovery data. Wallet recovery data includes:
+- The recovery phrase
+  - Type - BIP39, Electrum, Shamir Shares (SLIP39), AEZEED
+  - Passphrase (optional)
+- Derivation path
```

Comment:
As mentioned on the Google doc, I think we should be pushing for the use of descriptors for backing up of derivation paths / fingerprints / xpubs. 

See me and Stephen's discussion here: https://docs.google.com/document/d/1y19VskiZcm6AXO1fVECr2ze_xTehyz4GtGs4zzheKLU/edit?disco=AAAARAZp3lk

Reply:
What do some other people think?

On the one hand, I don't have anything against output descriptors. They seem like a very useful way of formatting data for many different types of outputs. It makes sense for there to be a common format many wallets understand.

On the other hand, I don't know that I want to get deep into engineering recommendations. With this section, we're proposing that developers and product designers think critically about their backup process -- that they should build it so that the user doesn't have to manually back stuff up. If the engineers want to encode their backup data inside of a JSON object, an output descriptor, or some other format, it could all result in the same good UX.

Perhaps I'm overthinking this one?

Reply:
Yep that suggestion looks great and love the template :)! Glad you got descriptor-pilled 😆 Chow and Thunder would of explained it a lot more eloquent that me. Yep any private key info should never touch a printer / go over the air in any way shape or form

Reply:
As you discussed in the Google Doc comments, descriptors are messy for users to write down, and if you want to avoid printers, then the QR code approach is also not useful (IMHO, the template should likely not contain any pre-filled information at all) .

I have a larger question here. Do we consider the focus of this page (and the Onboarding section in general) to be Lightning-first? If so, does the "piece of paper" backup method even have a place, since it can never be used for channel backups? If not, should we move that to a different section (an on-chain wallet case study)? Or is there a good reason to keep paper backups even if users still need to do a file-based backup for channels (ending up with two backup methods to keep track of)?

Reply:
@GBKS

> if you want to avoid printers, then the QR code approach is also not useful

To clarify, I am proposing using a descriptor QR which contains things like the derivation path, xPub, and fingerprint, but not a private key. Therefore, that could pass safely through a printer. Such a QR code would not replace the recovery phrase. The purpose would be so that the user does not need to do things like select their derivation path and verify the fingerprint.

> I have a larger question here. Do we consider the focus of this page (and the Onboarding section in general) to be Lightning-first? If so, does the "piece of paper" backup method even have a place, since it can never be used for channel backups?

That's a good question. For a mobile LN wallet, I'm always in favor of the cloud backup approach, so I think that should be considered the primary method of backup.

> Or is there a good reason to keep paper backups even if users still need to do a file-based backup for channels (ending up with two backup methods to keep track of)?

The only reason I can imagine why the paper wallet (or any form of manual backup) would be useful is if the user somehow loses access to their cloud backup. They could use the recovery phrase to restore on-chain funds, and if also reclaim their lightning funds after channels are force closed.

I acknowledge that's not an ideal user flow; I'm just saying it's possible. Is that a salient enough use-case to keep it here?

Reply:
Works for me. We just need to be clear about it.